Firebase/Auth/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Unreleased
```

Comment:
Should be v5.0.0. Weren't all or most of these cherry picked?

Reply:
Sure thing, wasn't sure if we wanted to put 5.0.0 or not since it's unreleased but makes sense. I'll change it!

Reply:
+1